nginx-wsgi-flask/docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+version: "3.7"
+services:
+  nginx-proxy:
+    build: nginx
+    restart: always
+    volumes:
+      - ./nginx/default.conf:/tmp/default.conf
+    environment: 
+      - FLASK_SERVER_ADDR=flask-app:8000
+    ports:
+      - "80:80"
+    links: 
```

Comment:
`links` field can be removed. By default the two services are placed in the same network so they can reach each other.